internal/pkg/flags/flags.go

```diff
@@ -85,8 +89,11 @@ var (
 //nolint:funlen // defines all program flags
 func DefineFlagVar() *FlagVar {
 	flagVar := new(FlagVar)
-	flag.StringVar(&flagVar.CertificateManagement, "cert-management", "", "Configures which certificate management"+
-		" system to use. Default is CNCF open source cert-manager. Accepted value: `gardener`")
+	flag.StringVar(&flagVar.CertificateManagement, "cert-management", certmanagerv1.SchemeGroupVersion.String(),
+		fmt.Sprintf("Configures which certificate management system to use. Accepted values: '%s', '%s'. Default: '%s'",
+			certmanagerv1.SchemeGroupVersion.String(),
+			gcertv1alpha1.SchemeGroupVersion.String(),
+			certmanagerv1.SchemeGroupVersion.String()))
```

Comment:
Proposing to use the SchemeGroupVersion instead as this is already defined with the package we are using and this also aligns with the value that is configured in management-plane-charts when switching between the two solutions.